src/main/java/org/jabref/logic/xmp/XmpUtilWriter.java

```diff
@@ -159,6 +170,38 @@ private static void writeDublinCore(PDDocument document,
         catalog.setMetadata(metadataStream);
     }
 
+    /**
+     * This method generates an xmp metadata string in dublin core format.
+     * <br/>
+     *
+     * @param entries   A list of entries, which are added to the dublin core metadata.
+     * @param xmpPreferences    The user's xmp preferences.
+     *
+     * @return  If something goes wrong (e.g. an exception is thrown), the method returns an empty string,
+     *          otherwise it returns the xmp metadata as a string in dublin core format.
+     */
+    public static String generateXmpString(List<BibEntry> entries, XmpPreferences xmpPreferences) {
+        XMPMetadata meta = XMPMetadata.createXMPMetadata();
+        for (BibEntry entry : entries) {
+            DublinCoreSchema dcSchema = meta.createAndAddDublinCoreSchema();
+            XmpUtilWriter.writeToDCSchema(dcSchema, entry, null, xmpPreferences);
```

Comment:
Is it possible to add an additional method `writeToDCSchema` with 3 parameters. We should avoid `null` parameters.